Copter: Failsafe refactor and FS_OPTIONS parameter documentation

Pedals2Paddles opened the pull request:
Starting to update the copter wiki failsafe pages to follow https://github.com/ArduPilot/ardupilot/pull/7432

Hwurzburg commented:
Matt, I have submitted a PR to your branch with my additions and edits and corrections...probably have guessed at one or two of the option  meanings incorrectly...but hopefully this will also serve as a basis for your updates to the other failsafe pages....thanks

Pedals2Paddles commented:
@Hwurzburg I did some more work on this today. I've updated the radio, gcs, and battery pages.  You were pretty close on the bitmask, so I updated that as well. I don't have this PC setup to render the pages right now, so I'm only editing the code and preying the formatting is right. I also haven't gone through spelling with a fine tooth comb yet either.  Please give this a quick run through if you have a moment as well.

Hwurzburg commented:
no problem...will build it and review it tomorrow...thank!

Pedals2Paddles commented:
I just ran the vagrant builder. Wow, much easier than the last time I tried to do this a few years ago.  So I'm fixing up a few more things now too.

Hwurzburg commented on copter/source/docs/radio-failsafe.rst:
```diff
@@ -4,149 +4,95 @@
--  Set the Failsafe Options to one of the three options:
+-  **Disabled** (Value 0) will disable the radio failsafe entirely.
+-  **Enabled Always RTL** (Value 1) will switch the copter to RTL Mode.  If the GPS position is not usable, the copter will change to Land Mode instead.
+-  **Enabled Continue with Mission in Auto Mode (Deprecated in 4.0+)** (Value 2) will ignore the failsafe in an Auto Mode mission. Otherwise, it will behave the same as *Enabled Always RTL*. This option no longer exists in ArduCopter 4.0. Instead, see the :ref:`FS_OPTIONS<FS_OPTIONS>` parameter for this function. Setting this value in Copter4.0 and later version will automatically be converted an ...
```
one addition needed...."will be modified to include bit (0) in the bitmask for "Continue if in AUTO Mode on Radio Failsafe".

Pedals2Paddles replied:
Good catch. I see a few spelling and grammar errorz in there too.  I'll do that this evening.

Pedals2Paddles commented:
Fixed up squashed and pushed!

Hwurzburg commented:
builds fine...LGTM

Pedals2Paddles commented:
Could someone kindly merge this?  The PR this covers in Copter 4 and master is in so this needs to go in.

Hwurzburg commented:
we a have review call tomorrow...it'll merge then unless someone has an issue with it...or Randy does it tonight, since he's the reviewer...

rmackay9 commented:
Merged, thanks!

proficnc commented:
Awesome!!!

Pedals2Paddles commented:
Thx!
